src/steps/os/linux.rs

```diff
@@ -458,10 +458,28 @@ pub fn run_pacdef(ctx: &ExecutionContext) -> Result<()> {
 
     print_separator("pacdef");
 
-    ctx.run_type().execute(&pacdef).arg("sync").status_checked()?;
+    let output = ctx.run_type().execute(&pacdef).arg("version").output_checked()?;
+    let string = String::from_utf8(output.stdout)?;
+    let new_version = string.contains("version: 1");
+
+    if new_version {
+        ctx.run_type()
+            .execute(&pacdef)
+            .args(["package", "sync"])
+            .status_checked()?;
+
+        println!();
+        ctx.run_type()
+            .execute(&pacdef)
+            .args(["package", "review"])
+            .status_checked()
+    } else {
+        ctx.run_type().execute(&pacdef).arg("sync").status_checked()?;
+
+        println!();
+        ctx.run_type().execute(&pacdef).arg("review").status_checked()
```

Comment:
This line should end with a `?;` and the return value of the function should be `Ok(())` but else it looks good.

Reply:
I'm not quite following. Your requested changes will result in semantically identical code.  Both arms of the `if` block end with the implicit return of `Result<()>`. Since there is no semicolon in 481 this is also the return value of the function.

Reply:
It will be semantically identical code but we use this style in the entire code base, which just makes it a bit easier to maintain. Just a style of code thingy.

Reply:
Thx